### ISSUE TITLE
Is it possible to let vosk model to recognize some common other than similar words.

### ISSUE THREAD

**Issue by ghost** [alphacep/vosk-api#600]
I want to implement Vosk in my program to recognize some command such as "refresh dashboard one", "show dashboard two".  But I found out that it is more likely to recognize as "show dashboard to" which should be two.  and also 'four' being recognized as 'for'.  Is there any way to adapt the model to recognize those words accurately?

**Comment by nshmyrev** [comment]
Yes, you can run online grammar (see test_words.py) or offline grammar adaptation (see adaptation page on our website).

**Comment by ghost** [comment]
The "Updating the language model" section of the adaptation page is only for Kaldi model right? The "Updating words and the vocabulary in the big models" section is to add new words to the model. I am planning to work on both the English and Chinese model. Which section of the adaptation page I should follow?

**Comment by ghost** [comment]
> The "Updating the language model" section of the adaptation page is only for Kaldi model right? The "Updating words and the vocabulary in the big models" section is to add new words to the model. I am planning to work on both the English and Chinese model. Which section of the adaptation page I should follow?



> offline grammar adaptation (see adaptation page on our website).

**Comment by nshmyrev** [comment]
> The "Updating the language model" section of the adaptation page is only for Kaldi model right

yes

**Comment by ghost** [comment]
I had tested the online grammar but it seems like it only works for English but not Chinese. 

Which section of the adaptation page I should follow to do offline grammar adaptation?

> yes
